openapi.yaml

```diff
@@ -17,1237 +21,222 @@ produces:
-        example: meal-plans
+        $ref: '#/definitions/PersonType'
       links:
-          $ref: '#/definitions/SelfLink'
+        $ref: '#/definitions/SelfLink'
       attributes:
+        type: object
         properties:
-          mealPlan:
+          firstName: &firstName
             type: string
-            description: Description of meal plan
-            example: Orange Rewards
-          balance:
-            type: number
-            format: float
-            description: Remaining balance on meal plan in USD
-            example: 23.8
-          lastUsedDate:
+            description: First name of person
+          lastName: &lastName
             type: string
-            format: date-time
-            description: The last time the meal plan was used
-            example: '2017-07-14T16:45:15Z'
-          lastUsedLocation:
+            description: Last name of person
+          middleName: &middleName
             type: string
-            description: Description of place the meal plan was last used
-            example: 'Java II'
-  NewAddressResultObject:
-    properties:
-      data:
-        $ref: '#/definitions/NewAddressResourceObject'
-  NewAddressResourceObject:
-    properties:
-      type:
-        $ref: '#/definitions/AddressType'
-      attributes:
-        required:
-        - addressType
-        - city
-        properties: &commonAddressAttributes
-          addressType:
+            description: Middle name of person
+          displayFirstName:
             type: string
-            description: Code for the type of address record.
-            example: CM
-          addressLine1:
+            description: Preferred first name
+          displayMiddleName:
             type: string
-            example: '123 Main Street'
-          addressLine2:
+            description: Preferred middle name
+          displayLastName:
             type: string
-            example: 'Apartment 123'
-          addressLine3:
+            description: Preferred last name
+          onid:
             type: string
-          addressLine4:
+            description: ONID username
+          email:
             type: string
-          houseNumber:
+            description: Primary email address
+          birthDate: &birthDay
             type: string
-          city:
+            description: Birthday of person
+          citizen: &citizen
             type: string
-          stateCode:
-            type: string
-            description: State code/abbreviation. This could also represent a province.
-            example: 'OR'
-          postalCode:
-            type: string
-            example: '97331'
-          countyCode:
+            enum: [FN, N, R, S, C]
+            description: |
+              Person's citizen code
+              - 'FN' - Foreign National
+              - 'N' - Non Resident Alien
+              - 'R' - Resident Alien
+              - 'S' - Substantial Presence Alien
+              - 'C' - Citizen
```

Comment:
I checked the [openapi file](https://github.com/osu-mist/students-api/blob/master/openapi.yaml#L88-L142) of Students API and looks like we are using this format:

```yaml
- `code` - description
```

Can you change the codes here and follow the same style? Also, can you add a section in our style guild for this as well?